Makefile

```
@@ -5,42 +5,42 @@ build: status=".status.build"
 build:
 	@echo '0' > ${status}
 	@$(foreach var,$(TARGETS), \
-		$(MAKE) -C $(var) $@; || echo '1' > ${status}; \
-		$(MAKE) -C $(var) -f Makefile.debian7 $@; || echo '1' > ${status}; \
-		$(MAKE) -C $(var) -f Makefile.debian8 $@; || echo '1' > ${status}; \
-		$(MAKE) -C $(var) -f Makefile.debian9 $@; || echo '1' > ${status}; \
-		$(MAKE) -C $(var) -f Makefile.debian10 $@; || echo '1' > ${status};)
+		$(MAKE) -C $(var) $@ || echo '1' > ${status} && \
+		$(MAKE) -C $(var) -f Makefile.debian7 $@ || echo '1' > ${status} && \
+		$(MAKE) -C $(var) -f Makefile.debian8 $@ || echo '1' > ${status} && \
+		$(MAKE) -C $(var) -f Makefile.debian9 $@ || echo '1' > ${status} && \
+		$(MAKE) -C $(var) -f Makefile.debian10 $@ || echo '1' > ${status})
```

I'd keep the `;` on each command just incase of fatal error the errors is reported in the command that fails otherwise make will report the whole line
```suggestion
		$(MAKE) -C $(var) $@ || echo '1' > ${status}; \
		$(MAKE) -C $(var) -f Makefile.debian7 $@ || echo '1' > ${status}; \
		$(MAKE) -C $(var) -f Makefile.debian8 $@ || echo '1' > ${status}; \
		$(MAKE) -C $(var) -f Makefile.debian9 $@ || echo '1' > ${status}; \
		$(MAKE) -C $(var) -f Makefile.debian10 $@ || echo '1' > ${status})
```